src/components/common/Map.css

```
@@ -1,4 +1,9 @@
 .os-map {
   height: 100vh;
   width: 80vw;
+  display: visible;
+
+  @media only screen and (max-width: 767px){
+    display: none;
+  }
```

You could invert the logic (which is a better option since we always try to code following the Mobile First workflow)
```suggestion
  display: none;

  @media screen and (min-width: 767px){
    display: block;
  }
```
BTW: I believe that `display: visible;` doest not exist